recipes/drflac/all/conanfile.py

```diff
@@ -10,15 +13,16 @@ class DrflacConan(ConanFile):
     topics = ("audio", "flac", "sound")
     license = ("Unlicense", "MIT-0")
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os", "compiler", "build_type", "arch"
+    settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False], 
         "fPIC": [True, False],
-        "buffer_size": "ANY",
+        "buffer_size": ["ANY"],
         "no_crc": [True, False],
         "no_ogg": [True, False],
         "no_simd": [True, False],
-        "no_stdio": [True, False]
+        "no_stdio": [True, False],
+        "no_wchar": [True, False]
```

Comment:
Is it a valid option on the previous version?

Reply:
No it was only added in the latest version. Should I add a check in `validate` and throw `ConanInvalidConfiguration`?

Reply:
It would be better to avoid false expectations is someone using `0.12.38` is switching that option and sees no change.  

Probably you can just remove the option for version `0.12.38`. It is just one line, no need to add anything to the `validate()` method.

Reply:
Thanks for the hints, addressed in 53925ca9643b0f3552b212bbe040fe4a6ad815dc.